src/dotty/tools/dotc/core/TypeComparer.scala

```diff
@@ -488,6 +488,9 @@ class TypeComparer(initctx: Context) extends DotClass with ConstraintHandling {
         case _ => tp2 isRef ObjectClass
       }
       compareJavaArray
+    case tp1: ExprType if ctx.phase.id >= ctx.gettersPhase.id =>
```

Comment:
Should this be `ctx.phase.id > ctx.gettersPhase.id` like in https://github.com/dotty-staging/dotty/blob/fix-%23938/src/dotty/tools/dotc/core/TypeComparer.scala#L455 ?